packages/client/ui/react-ui/vitest.config.ts

```diff
@@ -6,5 +7,8 @@ export default defineConfig({
         include: ["**/*.test.{ts,tsx}"],
         exclude: ["node_modules"],
         globals: true,
+        // This is needed because we are using the @ symbol to import from the src folder.
+        // Otherwise, Vitest will yell at us.
+        alias: [{ find: "@", replacement: resolve(__dirname, "./src") }],
```

Comment:
Not really in scope but it was annoying that auto imports were using the "@" and at the same time Vitest just wasn't having that so I figured i'll do everyone a favour and fix that.

Reply:
Can move into separate PR, @devlyn37 if you think is necessary